README.md

```diff
@@ -7,6 +7,50 @@ things, with as minimal dependencies as possible:
 1. Support just enough metadata to be able to look up deps.
 2. Do "the thing that pip does" when deciding what dist-info dir to look at.
 
+# Usage
+
+Example snippet to show how to get the metadata from a wheel. 
+
+```python
+from zipfile import ZipFile
+from metadata_please import basic_metadata_from_wheel
+
+zf = ZipFile('somepkg.whl')
+print(basic_metadata_from_wheel(zf, "somepkg"))
+```
+
+### Output
+
+```
+BasicMetadata(
+    reqs=[
+        'build',
+        'setuptools',
+        'pip',
+        'imperfect<1',
+        'tomlkit<1',
+        'click~=8.0',
+        'GitPython~=3.1.18',
+        'metatron==0.60.0',
+        'pkginfo~=1.9',
+        'pyyaml~=6.0',
+        'runez~=5.2',
+        'pathspec<1',
+        'virtualenv<20.21',
+        'tox~=3.28',
+        'requests~=2.27',
+        'urllib3~=1.26'
+    ],
+    provides_extra=frozenset(),
+    name='pynt',
+    requires_python='>=3.6',
+    url='https://stash.corp.netflix.com/projects/NFPY/repos/pynt/browse',
```

Comment:
Are you happy with this example?  Maybe use requests or something more recognizable instead?